includes/class-wp-job-manager-recaptcha.php

```diff
@@ -0,0 +1,236 @@
+<?php
+/**
+ * File containing the WP_Job_Manager_Recaptcha class.
+ *
+ * @package wp-job-manager
+ */
+
+namespace WP_Job_Manager;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * WP_Job_Manager_Recaptcha class.
+ *
+ * @since $$next-version$$
+ * @internal
+ */
+class WP_Job_Manager_Recaptcha {
+
+	/**
+	 * Stores static instance of class.
+	 *
+	 * @var WP_Job_Manager_Recaptcha The single instance of the class.
+	 */
+	private static $instance = null;
+
+	/**
+	 * Site key.
+	 *
+	 * @var string
+	 */
+	private $site_key;
+
+	/**
+	 * Secret key.
+	 *
+	 * @var string
+	 */
+	private $secret_key;
+
+	/**
+	 * The reCAPTCHA version.
+	 *
+	 * @var string
+	 */
+	private $recaptcha_version;
+
+	/**
+	 * Initialize class for landing pages.
+	 *
+	 * @since 2.0.0
+	 */
+	public function __construct() {
+		$this->site_key          = get_option( 'job_manager_recaptcha_site_key' );
+		$this->secret_key        = get_option( 'job_manager_recaptcha_secret_key' );
+		$this->recaptcha_version = get_option( 'job_manager_recaptcha_version' );
```

Comment:
Should we add class constants for these option names (and one for the default version)?

Reply:
Done in 260ba28f0540e2a73d48dda4b4bae2e99e3e56bf